cmd/v/help/init.txt

```diff
@@ -0,0 +1,4 @@
+Usage:
+  v init
+
+Setup the file structure for an already existing V project.
```

Comment:
please explain in a little more detail what files exactly will be created

Reply:
Okey c: